src/react/page/schedule/ScedulePage.tsx

```diff
@@ -36,24 +43,49 @@ export function EventCard(_props: EventCardProps) {
   return (
     <Card>
       <CardHeader
-        subheader={_props.event.location}
-        title={
-          <>
-            {_props.event.eventType === "Official" && (
-              <Box
-                component="img"
-                src="logo192.png"
-                sx={{
-                  alt: "",
-                  height: "auto",
-                  maxWidth: "20px",
-                }}
-              />
-            )}{" "}
-            <span>{_props.event.title}</span>
-          </>
-        }
+        subheader={<EventSubHeader event={_props.event} />}
+        title={<EventTitle event={_props.event} />}
       />
+      <CardContent>{_props.event.description}</CardContent>
     </Card>
   );
 }
+
+/**
+ * Renders the CardHeader subheader element. The event's start time and date
+ */
+function EventSubHeader(_props: EventCardProps) {
```

Comment:
Could you please desctructure the props in all three components in this file?

```suggestion
function EventSubHeader({event}: EventCardProps) {
```